tests/requirements.yml

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: MIT
+---
+
+- src: https://github.com/i386x/meta_test
+  version: ic-role
+  name: linux-system-roles.meta_test
```

Comment:
The test was not able to find this role in CI - is requirements.txt being processed by CI?

Reply:
Yes, I created PR for this: https://github.com/linux-system-roles/test-harness/pull/146